src/vs/workbench/contrib/debug/browser/repl.ts

```diff
@@ -305,6 +305,16 @@ export class Repl extends ViewPane implements IHistoryNavigationWidget {
 			`;
 
 			this.tree.rerender();
+
+			this.replInput.updateOptions({
+				fontSize,
```

Comment:
Why not take the `fontFamily`, `lineHeight` computed above? Why use `debugConsole.`

Reply:
Those values are for the inline css
- `lineHeight` computed above has type `string` (``? `${debugConsole.lineHeight}px` : '1.4em'``) and the one needed for `updateOptions` must be `number`
- `fontFamily` computed above resolves to `'var(--monaco-monospace-font)'` when the default font is requested

Reply:
Makes sense. Thank you